pkg/model/job_event.go

```diff
@@ -69,20 +66,16 @@ const (
 	// not hear back it will be stuck in reserving the resources for the job
 	JobEventInvalidRequest
 
+	// a job has been completed
+	JobEventCompleted
+
 	jobEventDone // must be last
 )
 
 // IsTerminal returns true if the given event type signals the end of the
 // lifecycle of a job. After this, all nodes can safely ignore the job.
 func (je JobEventType) IsTerminal() bool {
```

Comment:
is `JobEventInvalidRequest` also terminal?

Reply:
actually we are not using `JobEventInvalidRequest` any more. Will create a new PR to remove it along with other dead event types